server.R

```diff
@@ -1,10 +1,17 @@
+# Ensure the latest version of gt is installed
+if (!require("gt") || packageVersion("gt") < "0.3.0") {
+  install.packages("gt")
+}
```

Comment:
gt is currently running at `0.11.1`
```
packageVersion("gt")
[1] ‘0.11.1’
```
So where is this 0.3 version coming from?